src/modules/CRUD/components/Details/Header/Header.js

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import FitBoundButton from '../FitBoundButton';
 import DownloadButtons from '../DownloadButtons';
 
 const Header = ({ title, documents }) => (
-  <div className="details__header">
-    <h2 className="details__title">{title}</h2>
+  <header className="details__header">
```

Comment:
I would have taken the opportunity to make this component extensible by allowing arbitrary props to be applied on `<header />`.

*(Not mandatory at all!)*

Reply:
Ok it's a quick win, i pushed a newer version of it